incubator/hydra/templates/hydra_deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
if anything @kminehart I would consider removing postgres entirely from here, most people still aren't putting db's on k8s for a variety of reasons. Best method I have seen so far is the operator method, but still I think Hydra should just deploy agnostic of postgres, or use the postgres chart as a dependency